packages/udaru-core/lib/ops/validation.js

```diff
@@ -255,7 +255,8 @@ const teams = {
   },
   searchTeam: {
     organizationId: validationRules.organizationId,
-    query: requiredString
+    query: requiredString,
+    type: Joi.string().optional().allow('default', 'exact')
```

Comment:
the name default for the partial search doesn't sound well to me, maybe `pattern`?